packages/vsce/src/trees/CICSTree.ts

```diff
@@ -112,24 +104,21 @@ export class CICSTree implements TreeDataProvider<CICSSessionTree> {
           ),
           {
             ignoreFocusOut: true,
-            placeHolder: "Load Profile or Create New Profile",
+            placeHolder: "Edit Team Configuration File",
           }
         );
         if (profileNameToLoad) {
           // If Create New CICS Profile option chosen
-          if (profileNameToLoad.label.includes("\uFF0B")) {
-            if (configInstance.getTeamConfig().exists) {
-              // get all profiles of all types including zosmf
-              const profiles = configInstance.getAllProfiles();
-              if (!profiles.length) {
-                window.showErrorMessage("No profiles found in config file. Create a new config file or add a profile to get started");
-              }
-              const currentProfile = await ProfileManagement.getProfilesCache().getProfileFromConfig(profiles[0].profName);
-              const filePath = currentProfile?.profLoc.osLoc?.[0] ?? "";
-              await openConfigFile(filePath);
-            } else {
-              await this.createNewProfile();
-            }
+          if (profileNameToLoad.label.includes("\u270F")) {
+            // get all profiles of all types including zosmf
+            const profiles = configInstance.getAllProfiles();
+            const currentProfile =
+                profiles.length > 0 ? await ProfileManagement.getProfilesCache().getProfileFromConfig(profiles[0].profName) : null;
+            const filePath =
+                  currentProfile === null
+                    ? configInstance.getTeamConfig().opts.homeDir + "/zowe.config.json"
```

Comment:
did you notice if there was a constant in Zowe Explorer we could use for zowe.confgi.json? or perhaps even for the whole profile path? that would be nice.

Reply:
Is this something we can use? Or not the right thing?
https://github.com/zowe/zowe-explorer-vscode/blob/aea6d92717f4b765088a8d17e36e0c2c1a38a8d4/packages/zowe-explorer/src/extending/ZoweExplorerExtender.ts#L99

Reply:
Actually I reckon that is not that useful, it needs us to give the basic path and it just adds the filename. Probably not worth using.

Reply:
Resolving this thread..!!